src/WithdrawRequestNFT.sol

```diff
@@ -22,16 +28,19 @@ contract WithdrawRequestNFT is ERC721Upgradeable, UUPSUpgradeable, OwnableUpgrad
 
     uint32 public nextRequestId;
     uint32 public lastFinalizedRequestId;
-    uint96 public accumulatedDustEEthShares; // to be burned or used to cover the validator churn cost
+    uint16 public shareRemainderSplitToTreasuryInBps;
```

Comment:
We are changing the use of a storage variable here without an initialization function. Just need to make sure we call `updateShareRemainderSplitToTreasuryInBps` upon upgrade